src/main/java/group1/mips_simulator/SimController.java

```diff
@@ -1,14 +1,176 @@
 package group1.mips_simulator;
 
+import group1.mips_simulator.FrontEnd.Redraw;
+import group1.mips_simulator.components.Computer;
+import group1.mips_simulator.components.Value;
+import group1.mips_simulator.components.cpuParts.Register;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
 import javafx.fxml.FXML;
-import javafx.scene.control.Label;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
 
 public class SimController {
+
+    public Computer computer;
+    Redraw redraw = new Redraw();
+
+
+    public SimController(Computer c) {
+        this.computer = c;
+    }
+
+    //region Binary Displays
+
+    //region Left Column (General Purpose Registers)
+
+    public VBox VBoxLeft_GPRs;
+    public TextField GPR0;
+    public TextField GPR1;
+    public TextField GPR2;
+    public TextField GPR3;
+    public Button GPR0_button;
+    public Button GPR1_button;
+    public Button GPR2_button;
+    public Button GPR3_button;
+
+
+    //endregion
+
+    //region Middle Column (Index Registers)
+    public VBox VBoxCenter_IXRs;
+    public TextField IXR1;
+    public TextField IXR2;
+
+    public TextField IXR3;
+    public Button IXR1_button;
+    public Button IXR2_button;
+    public Button IXR3_button;
+
+
+    // endregion
+
+    //region Right Column (Other Registers)
+    public VBox VBoxRight_OtherRegs;
+    public TextField PC;
+    public TextField MAR;
+    public TextField MBR;
+    public TextField RI;
```

Comment:
im guessing RI is the equivalent of IR (Instruction Register) in spec right? 
may consider a refactor for all the "RI"'s to "IR"

Reply:
Yea it should be IR. This is a typo. Will refactor, good catch